### ISSUE TITLE
Tap not fired on element extending <a> in VoiceOver on iOS

### ISSUE THREAD

**Issue by lesliana** [Polymer/polymer#3952]
### Description
Tap event is not fired when you try to tap on an element that extends <a> using VoiceOver on iOS.

#### Live Demo
output.jsbin.com/faheleqiwo
Go to this jsbin on an iPhone.
Tap on the "hello world" link.
You see all events (touchstart, tap, touchend, mousedown, mouseup, click) correctly fired.
Turn on VoiceOver (Settings - General - Accessibility - VoiceOver).
Now return to the jsbin and tap on the "hello world" link again (in VoiceOver mode, you need to double tap to do this).
You only see touchstart, mousedown, mouseup, and click fired.
Try tapping again.
You only see mousedown, mouseup, and click fired.

This only repros for a native type extension element (<a is="x-a>) - it does not repro for other Polymer elements or for standard html elements like <a>.

#### Expected Results
In VoiceOver mode, I'd expect tap to be fired.

#### Actual Results
In VoiceOver mode, tap is not fired, so my tap handler is not hit and my element doesn't work.

### Browsers Affected
Repros on Safari and Chrome on iOS.  Does not repro on Android.

### Versions
- Polymer: v1.6.1

**Comment by lesliana** [comment]
Closing in favor of https://github.com/Polymer/polymer/issues/3957
